.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,27 @@
+# On each new tag deploy to npm
```

Comment:
This text is not clear enough, and can be easily misinterpreted.
Please replace it with the following:
"Every time a tag is pushed, it gets automatically published to npm".